Sources/Classes/core/Protocols/MiniAppUserInfoDelegate.swift

```diff
@@ -72,7 +72,7 @@ public extension MiniAppUserInfoDelegate {
 }
 
 /// Contact Object of a User
-public class MAContact: Codable, Equatable, Hashable, Identifiable {
+public struct MAContact: Codable, Equatable, Hashable, Identifiable {
```

Comment:
Any reason why this was changed to `struct ` from `class` ?

Reply:
This was changed as the class was having issues with the editing function, the bonded object was updated  and changed in the UI even before we tap on save or cancel buttons. So made it as struct to use it as pass by value.